openapi/paths/payout-requests@{id}@payment-instruments.yaml

```
@@ -0,0 +1,62 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Transactions
+  summary: Retrieve payout request's payment instruments
+  operationId: GetPayoutRequestPaymentInstruments
+  x-sdk-operation-name: getPaymentInstruments
+  description: |-
+    Retrieves a list of payment instruments with associated gateway of a payout request with specified ID.
+  responses:
+    200:
+      description: Payment instruments for the payout request retrieved successfully.
+      content:
+        application/json:
+          schema:
+            type: array
+            description: |-
+              List of available payment instruments per an associated gateway to offer for the payout.
+            items:
+              type: object
+              properties:
+                paymentInstrumentId:
+                  description: ID of the payment instrument to offer.
+                  type: string
+                gatewayName:
+                  description: |-
+                    Name of the payment gateway to use for the payout along the payment instrument.
+                    Empty for the requested payment instrument if it hasn't been used yet.
```

nitpick: suggestion for readability 
```suggestion
                    Name of the payment gateway and payment instrument to use for the payout.
                    This field is empty is the requested payment instrument has not been used yet.
```